src/web/assets/cp/src/Craft.js

```diff
@@ -95,3 +95,4 @@ import './js/UserIndex.js';
 
 // Custom elements
 import './js/CraftGlobalSidebar.js';
+import './js/CraftDisclosure.js';
```

Comment:
@brandonkelly I added this because the `Garnish.DisclosureMenu` looked to be very focused on menus specifically and I didn't see anything that was more generically "show or hide stuff" but let me know if I just missed it.